install_config/storage_examples/binding_pv_by_label.adoc

```diff
@@ -15,19 +15,19 @@ toc::[]
 == Overview
 This topic provides an end-to-end example for binding
 xref:../../architecture/additional_concepts/storage.adoc#persistent-volume-claims[persistent
-volume claims] (PVCs) to
+volume claims] (Persistent Volume Claims, PVCs) to
 xref:../../architecture/additional_concepts/storage.adoc#persistent-volumes[persistent
-volumes] (PVs) by defining labels in the PV and matching selectors in the PVC.
+volumes] (Persistent Volumes, PVs), by defining labels in the PV and matching selectors in the PVC.
```

Comment:
Aren't the full names being spelled out once at the beginning already? In the link text in the square brackets, e.g.:

```
xref:../../architecture/additional_concepts/storage.adoc#persistent-volume-claims[persistent volume claims] (PVCs)
```

Or am I misunderstanding?